client/webserver/site/src/js/app.js

```diff
@@ -154,14 +155,14 @@ export default class Application {
     pg.loader.style.backgroundColor = 'rgba(0, 0, 0, 0.5)'
     Doc.show(pg.loader)
     var hide
-    hide = e => {
-      if (!Doc.mouseInElement(e, pg.noteBox)) {
-        pg.noteBox.style.display = 'none'
-        unbind(document, hide)
+    hide = (el, e) => {
+      if (!Doc.mouseInElement(e, el)) {
+        el.style.display = 'none'
+        unbind(document, hide.bind(this, el))
       }
     }
     bind(pg.noteMenuEntry, 'click', async e => {
-      bind(document, 'click', hide)
+      bind(document, 'click', hide.bind(this, pg.noteBox))
```

Comment:
While `Function.bind` is used sparingly in the code, properly crafted arrow functions are much preferred. Here's one example of a way to do that.

```
var hide = (el, e, f) => {
  if (!Doc.mouseInElement(e, el)) {
    el.style.display = 'none'
    unbind(document, f)
  }
}
const hideNotes = e => { hide(this.noteBox, e, hideNotes) }
const hideProfile = e => { hide(this.profileBox, e, hideProfile) }
```